src/smart-components/common/form-renderer.js

```diff
@@ -5,16 +5,9 @@ import ReactFormRender from '@data-driven-forms/react-form-renderer/dist/cjs/for
 import componentMapper from '@data-driven-forms/pf4-component-mapper/dist/cjs/component-mapper';
 import FormTemplate from '@data-driven-forms/pf4-component-mapper/dist/cjs/form-template';
 
-import SummaryContent from '../workflow/add-groups/summary-content';
-
-const componentMapperExtended = {
-  ...componentMapper,
-  summary: SummaryContent
-};
-
 const FormRenderer = ({ formContainer, ...rest }) => (
   <ReactFormRender
-    componentMapper={ componentMapperExtended }
+    componentMapper={ componentMapper }
```

Comment:
is the approval using the whole pf4 mapper? We could save some space now that the wizard is no longer used. Its probably the biggest component from the bundle.